release/azure-pipelines-release.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
 - job: build_release
   displayName: "Create the OpenTitan release"
-  timeoutInMinutes: 360
+  timeoutInMinutes: 500
```

Comment:
If we don't have an issue already, please create an issue to investigate the CI release pipeline to see if we can optimize the release process and cut this time down.

Reply:
Issue created: #18991